code/modules/mob/living/carbon/human/human_dummy.dm

```diff
@@ -82,6 +82,17 @@ GLOBAL_LIST_EMPTY(dummy_mob_list)
 	ADD_TRAIT(src, TRAIT_IMMOBILIZED, TRAIT_SOURCE_TUTORIAL)
 	anchored = TRUE
 
+/mob/living/carbon/human/realistic_dummy/Initialize() // Now comes pre-fitted with Marine gear!! Tutorial realism increased by 400%!!!
+	. = ..()
+	status_flags |= FAKESOUL
+	create_hud()
+	arm_equipment(src, /datum/equipment_preset/uscm/tutorial_rifleman)
+
+/mob/living/carbon/human/realistic_dummy/updatehealth()
+	. = ..()
+	if((round(health * 100 / species.total_health, 10)) >= 70)
```

Comment:
if (round( x * 100 / 100, 10) >= 70. Like why though. 
(x * 100)/100 = y is just x = y, it does nothing, then you round it to the nearest multiple of 10 for some reason, then you check if it's over 70.